sections/security/requestpayloadsizelimit.md

```diff
@@ -0,0 +1,60 @@
+# Limit payload size using a reverse-proxy or a middleware
+
+### One Paragraph Explainer
+
+When handling incoming requests in your web application, you should limit the size of their respective payloads. Incoming requests with
+unlimited body/payload sizes can lead to your application crashing due to a denial-of-service outage or other unwanted side-effects.
+Many popular middleware-solutions for parsing request bodies, such as the already-included `body-parser` package for express, expose
+options to limit the sizes of request payloads, making it easy for developers to implement this functionality. You can also
+integrate a request body size limit in your reverse-proxy/web server software if supported. Below are examples for limiting request sizes using
+`express` and/or `nginx`.
+
+### Example code for `express`
+
+```javascript
+const express = require('express');
+
+const app = express();
+
+const parseJsonBody = express.json({ limit: '300kb' }); // body-parser defaults to a body size limit of 100kb
+const handleErrors = (err, req, res, next) => res.sendStatus(400); // simple express middleware to handle body parsing error
```

Comment:
@BrunoScheufler you sure body parser won't return 400 anyway?

Reply:
Just noticed that this is truly redundant. I actually can't reckon why this didn't work in the first run, thanks!